docs/overview/README.md

```diff
@@ -1,18 +1,27 @@
 # Overview
 
-The IOTA client library is a stateless library that provides a high level abstractions on top of IOTA node software to help interact with IOTA network in user friendly way and allows you to do the following, for instance:
+To communicate with the IOTA network, you have to connect and interact with a [node](https://chrysalis.docs.iota.org/node-software/node-software.html). The iota.rs library wraps routine tasks and minute details into convenient API calls. It will generate addresses for you, send messages, sign and send transactions, and more.
 
-- Create messages and transactions
-- Sign transactions
-- Generate addresses
-- Interact with an IOTA node
+Beyond establishing the initial connection to a node, iota.rs has no state. Operations use only the data that you pass during a call and have no effect on your software beyond returning a value. You are in full control of the data flow in your application.
```

Comment:
```suggestion
Beyond establishing the initial connection to a node, iota.rs has no state. Operations only use the data that you pass during a call and have no effect on your software beyond returning a value. You are in full control of the data flow in your application.
```

Reply:
Charles had the same suggestion and we discussed it in Slack DMs :sweat_smile: I keep it this way to avoid ambiguity (again): "operations just use data and don't do anything else with it" vs "operations use nothing but data"